example/example-command/example-command-api/src/main/java/uk/gov/justice/services/example/cakeshop/command/api/MakeCakeCommandApi.java

```diff
@@ -0,0 +1,28 @@
+package uk.gov.justice.services.example.cakeshop.command.api;
+
+import org.slf4j.Logger;
+import uk.gov.justice.services.core.annotation.Handles;
+import uk.gov.justice.services.core.annotation.ServiceComponent;
+import uk.gov.justice.services.core.sender.Sender;
+import uk.gov.justice.services.messaging.Envelope;
+
+import javax.inject.Inject;
+
+import static org.slf4j.LoggerFactory.getLogger;
+import static uk.gov.justice.services.core.annotation.Component.COMMAND_API;
+
+@ServiceComponent(COMMAND_API)
+public class MakeCakeCommandApi {
+
+    private static final Logger LOGGER = getLogger(MakeCakeCommandApi.class);
+
+    @Inject
+    Sender sender;
+
+    @Handles("cakeshop.commands.make-cake")
+    public void handle(final Envelope envelope) {
+        LOGGER.info("=============> Inside make-cake Command API");
```

Comment:
We haven't really implemented make-cake yet.  Ony add-recipe.  Handlers have been created just to ensure it doesn't crash.